test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -531,7 +531,7 @@ public void ValidateFunctionName_DoesNotThrowOnValidName(string functionName)
         [Theory]
         [InlineData(null)]
         [InlineData("")]
-        [InlineData("_binding")]
+        [InlineData("__binding__")]
```

Comment:
Per discussion with python worker team, we want to disallow names with double underscore.

Reply:
Your regex disallows __ anywhere, so perhaps you should add cases:

binding
bind__ing
binding